src/analyses/goto_rw.cpp

```diff
@@ -621,6 +621,17 @@ void rw_range_sett::get_objects_rec(const typet &type)
   }
 }
 
+void rw_range_sett::get_array_objects(
+  const irep_idt &,
+  goto_programt::const_targett,
+  get_modet mode,
+  const exprt &pointer)
+{
+  object_descriptor_exprt ode;
+  ode.build(dereference_exprt{skip_typecast(pointer)}, ns);
+  get_objects_rec(mode, ode.root_object(), -1, -1);
```

Comment:
🪄 magic numbers?

Reply:
Fair point! These are used throughout `goto_rw`, so I'll post a follow-up PR that fix this throughout this piece of code.